### PR TITLE
Profile activity

### PR DESCRIPTION
**Pull request by san-coding**
Added profile activity and added an `intent` to `loginBtn.setOnclicklistener()` to call profile activity

### CODE REVIEW

**Comment by sudhay23** [comment]
Great. it works. just reduce the font size a bit

File: androidApp/MovieRecommendationApp/app/src/main/java/com/example/movierecommendationapp/LoginFragment.java [hunk 1]
```diff
@@ -33,13 +34,21 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
...
+        });
+
         clearBtn.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
                 AlertDialog.Builder alertDialog = new AlertDialog.Builder(getActivity());
                 alertDialog.setTitle("Clear Fields");
                 alertDialog.setMessage("Are you sure to clear?");
-                alertDialog.setIcon(R.drawable.ic_baseline_warning_24);
+                alertDialog.setIcon(R.drawable.ic_launcher_foreground);
```
Comment (sudhay23) [inline androidApp/MovieRecommendationApp/app/src/main/java/com/example/movierecommendationapp/LoginFragment.java:51]: Revert the icon back to `ic_baseline_warning_24`

**Review by sudhay23 — CHANGES_REQUESTED** [review]

**Comment by san-coding** [comment]
@sudhay23  I have added update profile feature too, check it out

File: androidApp/MovieRecommendationApp/app/src/main/java/com/example/movierecommendationapp/LoginFragment.java (hunk 1 above)
Comment (san-coding) [reply androidApp/MovieRecommendationApp/app/src/main/java/com/example/movierecommendationapp/LoginFragment.java:51]:
```suggestion
                alertDialog.setIcon(R.drawable.ic_launcher_foreground);
```
Committed the above

**Pull request merged**